### PR TITLE
Fixed two issues

### PR DESCRIPTION
**Pull request by jumlu**
- Changed return of State from "PowerSwitchService.State" to "boolean" (required for HASS to work as light)
- Removed PowerSwitchProgramService, as always "None".

### CODE REVIEW

File: .gitignore
Comment (tschamm) [inline .gitignore:1]: either remove dimmertest.py from examples directory, or keep it in and commit changes.

File: boschshcpy/models_impl.py [hunk 1]
```diff
@@ -1350,7 +1348,6 @@ def child_lock(self, state: bool):
     
     def update(self):
         self._powerswitch_service.short_poll()
-        self._powerswitchprogram_service.short_poll()
         self._communicationquality_service.short_poll()
         self._childprotection_service.short_poll()
         super().update()
```
Comment (tschamm) [inline boschshcpy/models_impl.py:1353]: You can remove the `update()` method, this is part of parent class `SHCDevice`.

File: boschshcpy/models_impl.py [hunk 2]
```diff
@@ -1322,13 +1321,12 @@ class SHCMicromoduleDimmer(SHCLight):
...
         self._powerswitch_service = self.device_service("PowerSwitch")
-        self._powerswitchprogram_service = self.device_service("PowerSwitchProgram")
         self._communicationquality_service = self.device_service("CommunicationQuality")
         self._childprotection_service = self.device_service("ChildProtection")
 
     @property
-    def state(self) -> PowerSwitchService.State:
-        return self._powerswitch_service.value
+    def state(self) -> bool:
+        return (self._powerswitch_service.value == self.PowerSwitchService.State.ON)
```
Comment (tschamm) [inline boschshcpy/models_impl.py:1329]: Ok, as a light switch is returning bool values on HA side.

**Review by tschamm — COMMENTED** [review]
Can you implement the requested changes

**Review by tschamm — CHANGES_REQUESTED** [review]
Need to address changes before merging

File: boschshcpy/models_impl.py (hunk 1 above)
Comment (jumlu) [reply boschshcpy/models_impl.py:1353]: The whole update() method? All other models define an update method?

File: .gitignore
Comment (jumlu) [reply .gitignore:1]: removed.

File: boschshcpy/models_impl.py (hunk 1 above)
Comment (tschamm) [reply boschshcpy/models_impl.py:1353]: In the current master, only the base class provides the update() method.

Comment (jumlu) [reply boschshcpy/models_impl.py:1353]: Ah, good idea. Removed.

**Comment by jumlu** [comment]
Changes implemented.

**Review by tschamm — APPROVED** [review]
LGTM

**Pull request merged**